tensorflow/python/keras/utils/__init__.py

```diff
@@ -20,6 +20,7 @@
 
 from tensorflow.python.keras._impl.keras.utils.data_utils import GeneratorEnqueuer
 from tensorflow.python.keras._impl.keras.utils.data_utils import get_file
+from tensorflow.python.keras._impl.keras.utils.data_utils import OrderedEnqueuer
```

Comment:
Shouldn't there be an associated unit test for this change?

Reply:
Tests are already there, starting here https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/_impl/keras/utils/data_utils_test.py#L204 ... this is just enabling the API in `tf.keras`.